src/main/scala/io/qbeast/spark/delta/StagingDataManager.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Qbeast Analytics, S.L.
+ */
+package io.qbeast.spark.delta
+
+import io.qbeast.core.model.QTableID
+import io.qbeast.core.model.RevisionUtils.isStagingFile
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.delta.DeltaLog
+import org.apache.spark.sql.delta.actions.{AddFile, RemoveFile}
+import org.apache.spark.sql.{DataFrame, Dataset, SparkSession}
+
+case class StagingDataManager(tableID: QTableID) {
```

Comment:
Scala Doc is missing in this class and methods.

Reply:
Done